src/Window/Context.cs

```diff
@@ -10,7 +9,7 @@ namespace SFML.Window
     /// This class defines a .NET interface to an SFML OpenGL Context
     /// </summary>
     //////////////////////////////////////////////////////////////////
-    public class Context : CriticalFinalizerObject
```

Comment:
CriticalFinalizerObject is necessary last I checked to ensure cleanup of OpenGL contexts.